seedlet/assets/css/style-editor.css

```diff
@@ -283,7 +283,6 @@ object {
 
 .wp-block-a8c-blog-posts .entry-title a:hover {
 	color: var(--global--color-primary-hover);
-	text-decoration: underline;
```

Comment:
Looks like the issue happens both in the front-end and editor. Won't this change only fix it for the editor?

Reply:
Oh whoops — that was an oversight. I'll update. 👍